barretenberg/ts/src/barretenberg/index.ts

```diff
@@ -114,6 +118,39 @@ export class Barretenberg extends BarretenbergApi {
   getWasm() {
     return this.wasm;
   }
+
+  getBbApi(): BbApiBase {
+    return this.bbApi;
+  }
+
+  // Wrap ClientIVC methods used by AztecClientBackend
+  async clientIvcStart(command: Parameters<BbApiBase['clientIvcStart']>[0]) {
+    return this.bbApi.clientIvcStart(command);
+  }
```

Comment:
I'll revisit the best way to live with two binding generations - for now, the Barretenberg wrapper exposes both